package.json

```diff
@@ -25,7 +25,7 @@
     "activationEvents": [
         "onLanguage:c",
         "onLanguage:cpp",
-        "onLanguage:cuda",
+        "onLanguage:cuda-cpp",
```

Comment:
This feature is available in vscode 1.56.0 or later. We need to bump the required vscode version (on Line 11).

Reply:
Done, I also updated the package @types/vscode package to match the this version.